api/router.go

```diff
@@ -248,6 +248,10 @@ func NewRouter(s *Server) *mux.Router {
 	r.Path("/v1/ws/backingimages").Handler(f(schemas, backingImageStream))
 	r.Path("/v1/ws/{period}/backingimages").Handler(f(schemas, backingImageStream))
 
+	backupBackingImageStream := NewStreamHandlerFunc("backupbackingimages", s.wsc.NewWatcher("backupBackingImage"), s.backupBackingImageList)
+	r.Path("/v1/ws/backingimages").Handler(f(schemas, backupBackingImageStream))
```

Comment:
Hi @ChanYiLin , 
Here should be

```
r.Path("/v1/ws/backupbackingimages"
r.Path("/v1/ws/{period}/backupbackingimages")
``` 
?

Otherwise, there is no endpoint support `/v1/ws/backupbackingimages`.
<img width="1496" alt="Screenshot 2024-07-07 at 6 16 21 PM" src="https://github.com/longhorn/longhorn-manager/assets/5744158/42d6b681-5692-4017-abbe-634f4a04a576">

Reply:
oh you  are right